doc/api/deprecations.md

```diff
@@ -511,6 +511,14 @@ Type: Documentation-only
 The `fs.SyncWriteStream` class was never intended to be a publicly accessible
 API.
 
+<a id="DEP0062"></a>
+### DEP0062: node --debug
+
+Type: Runtime
+
+`--debug` activates the legacy V8 debugger, which has been removed as of V8 5.8.
```

Comment:
Is it worth being specific about V8 removing the JSON API and not the debugger _per se_? Or would that just be more confusing? (And am I even correct about that? :-P )